light-system-programs/programs/user_registry/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+use anchor_lang::prelude::*;
+
+declare_id!("6UqiSPd2mRCTTwkzhcs1M6DGYsqHWd5jiPueX3LwDMXQ");
+
+const USER_ENTRY_SEED: &[u8] = b"user-entry";
+
+#[program]
+pub mod user_registry {
+    use super::*;
+
+    pub fn initialize_user_entry(
+        ctx: Context<InitializeUserEntry>,
+        light_pubkey: [u8; 32],
+    ) -> Result<()> {
+        let mut user_entry = ctx.accounts.user_entry.load_init()?;
+        user_entry.solana_pubkey = ctx.accounts.signer.key().to_bytes();
+        user_entry.light_pubkey = light_pubkey;
+
+        Ok(())
+    }
+}
+
+#[account(zero_copy)]
+pub struct UserEntry {
+    pub solana_pubkey: [u8; 32],
+    pub light_pubkey: [u8; 32],
```

Comment:
We should either make this 64 bytes or add another 32 bytes field with the encryption pubkey.
You can get the encryption pubkey from the account.
```
let seed32 = bs58.encode(new Uint8Array(32).fill(1));
let k0 = new Account({ poseidon, seed: seed32 });

k0.encryptionKeypair.publicKey
k0.pubkey
// as string
let pubKey = k0.getPublicKey();
```

Reply:
Done